### DISCUSSION TITLE
Shelly wave 1 config misalignment

### DISCUSSION THREAD

**Discussion by anto35** [zwave-js/zwave-js#7132] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

On both Shelly Wave 1 [QPSW-0A1X16EU] and Shelly Wave 1 mini [QMSW-0A1X8EU] (at least), when I select 10 ms for parameter 25[O1: Auto-On/Off Timer Unit], it appears that the parameter value seems to be interpreted as milliseconds steps.

What did you expect to happen instead?

According to The XML from [Certification Database](https://products.z-wavealliance.org/Products/5025/XML), the description of parameter 25 is "Set timer units to Seconds or Milliseconds for O1" and the description is "0 seconds, 1 millisecods".

I don't really know if it's common for all Shelly wave devices or just these ones, but at least these two devices are impacted.

### Software versions

Driver (node-zwave-js): 13.1.0

Z-Wave JS UI: 9.17.0

### Device information

Manufacturer: Shelly Qubino
Model name: Wave 1 / Wave 1 mini

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
@QubinoHelp ?

**Reply by QubinoHelp** [reply]
Hi @anto35 & @AlCalzone 

We apologize for the oversight on our part and will make the necessary corrections to ensure everything is properly configured. We need to add two products to the configuration file, and we will make this change simultaneously.

As specified in the Knowledge Base, the resolution is 100 ms. You can currently input values in milliseconds, but please note that any value lower than this will not have the desired effect. For example, inputting 500 will result in a half-second delay. At the moment, the firmware is set to have steps in milliseconds, but we plan to address and improve this in a future update.
